### PR TITLE
Add tc talk to readme

### PR DESCRIPTION
**Pull request by pvanderknyff**
Adds a link to the youtube recording of mine and Stephen's tc talk to the Getting Started section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ At Tableau, we pride ourselves in helping people see and understand their data..
 
 Review the [Tableau Connector SDK developer guide](https://tableau.github.io/connector-plugin-sdk/) to help you design, build, and test your connector.
 
+You can also watch our [Tableau Conference 2019 talk](https://www.youtube.com/watch?v=_rfQtHLWWxU), where two of our developers walk through the basics of creating a custom Tableau connector and pacaking it into a single .tcao file.
```
Comment (lukewrites) [inline README.md]:
```suggestion
You can also watch our [Tableau Conference 2019 talk](https://www.youtube.com/watch?v=_rfQtHLWWxU), where two of our developers walk through the basics of creating a custom Tableau connector and pacaking it into a single `.taco` file.
```

Comment (lukewrites) [reply README.md]: hard to read the suggestion; there's a typo

File: README.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ At Tableau, we pride ourselves in helping people see and understand their data..
 
 Review the [Tableau Connector SDK developer guide](https://tableau.github.io/connector-plugin-sdk/) to help you design, build, and test your connector.
 
+You can also watch our [Tableau Conference 2019 talk](https://www.youtube.com/watch?v=_rfQtHLWWxU), where two of our developers walk through the basics of creating a custom Tableau connector and pacaking it into a single `.taco` file.
```
Comment (lriggs) [inline README.md]: pacaking  -> packaging

Comment (pvanderknyff) [reply README.md]: Fixed

**Pull request merged**